k8s_dp/k8s_dp.p4

```diff
@@ -501,6 +574,247 @@ control k8s_dp_control(
 		const default_action = NoAction();
 	}
 	
+        action acl_deny() {
+            pkt_cntr.count(80);
+            drop_packet();
+        }
+
+        action set_range_check_ref (AclPolicyId_t pol_id,
+                                    RangeCheckRefType_t range_check_ref) {
+            meta.acl_status = MATCH_RULE;
+            meta.acl_pol_id = pol_id;
+            meta.range_check_ref = range_check_ref;
+            pkt_cntr.count(42);
+        }
+
+        action set_status_match_ipset_only (AclPolicyId_t pol_id) {
```

Comment:
are we not changing pol_id to ipsetidx?

Reply:
This current set of changes are limited to the ones where InfraManager control code is mostly same for the both targets. This parameter does not affect that API interface. But, I agree, we should converge on these things as well for clarify but it can be done later.